index.html

```diff
@@ -26,7 +26,7 @@
             <h1 class="header-title">Feeds</h1>
         </div>
 
-        <div class="menu">
+        <div class="slide-menu hidden">
```

Comment:
Can you remove the `hidden` class from this element? That class does not exist in the CSS file and was removed in prior commits.